requirements/local.txt

```diff
@@ -1,6 +1,5 @@
 -r base.txt
 
-Werkzeug[watchdog]==2.2.3 # https://github.com/pallets/werkzeug
```

Comment:
Removed the requirement so the version required for G2P is installed automatically